chsdi/templates/htmlpopup/bauzonen.mako

```diff
@@ -1,14 +1,17 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
-    <tr><td class="cell-left">${_('nutzung')}</td>
-      % if lang in ('de', 'rm', 'en'):
-           <td>${c['attributes']['nutz_de'] or '-'}</td>
-      % else:
-           <td>${c['attributes']['nutz_fr'] or '-'}</td>
-      % endif
-    </tr>
-    <tr><td class="cell-left">${_('gemeinde')}</td>    <td>${c['attributes']['name']}</td></tr>
-    <tr><td class="cell-left">${_('kanton')}</td>    <td>${c['attributes']['kt_kz'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('flaeche_m2')}</td>    <td>${int(round(c['attributes']['flaeche_qm'])) or '-'}</td></tr>
+  <tr><td class="cell-left">${_('tt_bauzonen_CH_Code_HN')}</td>    <td>${c['attributes']['ch_code_hn'] or '-'}</td></tr>
+  <tr><td class="cell-left">${_('tt_bauzonen_CH_BEZ_D')}</td>
+    % if lang in ('de', 'rm', 'en'):
+      <td>${c['attributes']['ch_bez_d'] or '-'}</td>
+    % else:
+      <td>${c['attributes']['ch_bez_f'] or '-'}</td>
+    % endif
+  </tr>
+	<tr><td class="cell-left">${_('tt_bauzonen_gemeindetypen_BFS_NO')}</td>    <td>${c['attributes']['bfs_no'] or '-'}</td></tr>
```

Comment:
remember the convention for translation of attributes? When you fill up the tooltip table it should add the translations automatically to the translations table (and remove them automatically when you remove the layer)

Reply:
but as it is an old layer, this should be ok.